### PR TITLE
Update metricflow-commands.md

### PR DESCRIPTION
**Pull request by mirnawong1**
add dbt list saved queries command and  --show-exports flag per recent eng update: https://linear.app/dbt-labs/issue/SL-1655/cloud-cli-list-exports-in-dbt-sl-list-saved-queries#comment-aae9ef8f

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/static/_redirects`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/build/metricflow-commands.md [hunk 1]
```diff
@@ -174,6 +175,34 @@ Options:
...
+You can also add the `--show-exports` flag (or option) to show each export listed under a saved query:
+
+```bash
+dbt sl list saved-queries --show-exports
+```
+
+**Output**
+
+```bash
+[*main}[~/git/jaffle-sl-template]$ ../dbti/dbt sl list saved-queries --show-exports
```
Comment (WilliamDee) [inline website/docs/docs/build/metricflow-commands.md]: lets get rid of most of this and just leave 
```
dbt sl list saved-queries --show-exports
```

File: website/docs/docs/build/metricflow-commands.md [hunk 2]
```diff
@@ -174,6 +175,34 @@ Options:
...
+WARNING: version 0.35.23 of the dbt Cloud CLI is now available. During beta, old versions are not supported.
```
Comment (WilliamDee) [inline website/docs/docs/build/metricflow-commands.md]: Remove this line

File: website/docs/docs/build/metricflow-commands.md [hunk 3]
```diff
@@ -174,6 +175,34 @@ Options:
...
+
+The list of available saved queries:
+- new_customer_orders
+  exports:
+       - Export(name_customer_orders, exportAS=TABLE)
+       - Export(name_customer_orderss, exportAS=VIEW)
+       - Export(name_customer_ordersss, alias=test, schemas=lfg, exportAS=TABLE)
```
Comment (WilliamDee) [inline website/docs/docs/build/metricflow-commands.md]: Let's change the namings here, maybe
```
- Export(new_customer_orders_table, exportAs=TABLE)
- Export(new_customer_orders_view, exportAs=VIEW)
- Export(new_customer_orders, alias=orders, schemas=customer_schema, exportAs=TABLE)
```

**Review by WilliamDee — COMMENTED** [review]
Few recommended changes, otherwise looks good!

**Comment by mirnawong1** [comment]
thanks soo much @WilliamDee ! i've folded in your changes and i hope it's ok!

**Review by WilliamDee — APPROVED** [review]
Thank you!! 🙏

**Pull request merged**